lib/models/user/requests.js

```diff
@@ -66,7 +66,9 @@ module.exports = function(UserSchema) {
                             requestee: user
                         }
                     });
-
+                    const fullName = this.firstName + " " + this.lastName;
+                    const pushMessage = fullName + " " + "requested a share";
+                    user.sendPushNotification({title: "New Sharing Request", body: pushMessage});
                     callback(null, request);
```

Comment:
Is there any way to avoid sending the text here and instead letting the client pick the language